ansible/roles/webserver-content/tasks/main.yml

```diff
@@ -37,6 +37,21 @@
   args:
     chdir: /tmp/connectbox_client/{{ connectbox_client_path }}
 
+- name: retrieve connectbox-reports-repo
```

Comment:
I realise this is the pattern we’ve used in the past, but I can’t remember why we’re doing it as opposed to creating the directory and cloning directly into that new directory. I’d prefer to clone into the new directory - can you make that change if you can’t think of a good reason to keep it like this? (I’m happy to apply the pattern to our other clones as a part of a cleanup)